src/Pages/AddNewAds/AddNewAds.tsx

```diff
@@ -65,9 +67,9 @@ const AddNewAds = () => {
       const initActiveValue =
         roomDetails?.isActive !== undefined && String(roomDetails?.isActive);
       setIsActive(initActiveValue);
-
-      setValue("discount", roomDetails?.room?.discount);
-
+      setValue("room", "555");
```

Comment:
I don't understand the reason for setting the room value manually, could you explain the reason behind that?

Reply:
i tried to check how to set default value to solve MUI error [in select field] but i forgot to remove it before push code